### PR TITLE
[PL] translation of control structures

### PR DESCRIPTION
**Pull request by Koziolek**
in scope od issue #202

### CODE REVIEW

**Comment by doomspork** [comment]
Polish will be done in no time at this pace @Koziolek, thank you very much for all of your help!  👍 

@kdebowski would you do the honors? :D

**Comment by Koziolek** [comment]
Change name of PR.

File: pl/lessons/basics/control-structures.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+iex> case :even do
+...>   :odd -> "Odd"
+...>   _ -> "Not Odd"
+...> end
+"Not Odd"
+```
+
+Konstrukcja `_` działa tak samo, jak `else`, czyli dopasuje "wszystko inne".
+
+Jako że `case` wykorzystuje dopasowanie wzorców, a zatem wszystkie zasady tam obowiązujące są zachowane.  Jeżeli chcesz dopasować istniejącą zmienną, to musisz użyć operatora `^`:
```
Comment (kdebowski) [inline pl/lessons/basics/control-structures.md]: I think "a zatem" is unwanted here. The sentence:

> Jako że `case` wykorzystuje dopasowanie wzorców, wszystkie zasady tam obowiązujące są zachowane.

is sufficient.

File: pl/lessons/basics/control-structures.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+...>   true -> "Catch all"
+...> end
+"Catch all"
+```
+
+## `with`
+
+Konstrukcja `with` jest to forma, którą możemy użyć zamiast zagnieżdżonych wyrażeń `case` albo w sytuacji, gdy nie mogą być one powiązane z jednoznaczny sposób. Wyrażenie `with` składa się ze słowa kluczowego, generatora i wyrażenia.
+
+Zajmiemy się jeszcze generatorami przy okazji omawiania list generowanych, a na chwilę obecną jedyne co musimy wiedzieć to, że używają dopasowania wzorców, by połączyć elementy po prawej stronie `<-` z tymi po lewej.
```
Comment (kdebowski) [inline pl/lessons/basics/control-structures.md]: I suppose "list generowanych" should be "list składanych". Like in Python we have list comprehensions (listy składane, składanie list).

**Comment by kdebowski** [comment]
@Koziolek I added two notes

File: pl/lessons/basics/control-structures.md (hunk 2 above)
Comment (Koziolek) [inline pl/lessons/basics/control-structures.md]: I don't know python to polish translation. Technically `List comprehension` has not equivalent in polish lang. Give me 10 minutes to fix that two

**Comment by Koziolek** [comment]
Fixed & merged

**Pull request merged**